guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.md

```diff
@@ -0,0 +1,161 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: Multiselect widget
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+## Overview
```

Comment:
This heading is unnecessary. The first paragraph of a topic is generally accepted as an overview.